conf/namespaced-wireguard-vpn.conf

```diff
@@ -20,6 +20,15 @@ WIREGUARD_ALLOWED_IPS=0.0.0.0/0,::0/0
 # interface
 WIREGUARD_IP_ADDRESSES=10.0.0.1/32,fd12:3456:789a:1::1/128
 
+# Assuming a sane VPN provider:
+# IPv4: 1440
+# IPv6: 1420
+# If using PPPoE(typically DSL) -=8
+WIREGUARD_INITIAL_MTU=1420
```

Comment:
I don't know a heck of a lot about MTU settings. What would go wrong if the default we picked is too high for someone's VPN? Would it fall back to a reasonable setting, or would it just not connect? If the latter, we should probably not pick a default and only set MTU if the user positively sets a value here.

Reply:
Well, I require an initial MTU to be set to be able to reduce it at every step for the multi-layer case. We could be paranoid and go with something smaller, 1280 is considered absolutely universally compatible even on mobile networks to my knowledge. I'm not sure what happens by default if no MTU is set. A fair compromise could be 1412, DSL is still fairly common in many places after all.

Reply:
I've done a little more reading, and it looks like Wireguard defaults to 1420, so I guess that's reasonable for us too. I just didn't want to pick a value greater than what we're getting by default in the one layer case.